pkg/operator/controller/nodeobservability/daemonset.go

```diff
@@ -14,6 +14,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 
 	v1alpha2 "github.com/openshift/node-observability-operator/api/v1alpha2"
+	opctrl "github.com/openshift/node-observability-operator/pkg/operator/controller"
```

Comment:
I think we can keep the same name for the controller (or any other package) wherever we are importing it, just for easy readability.

Reply:
From what I can see it's `opctrl` everywhere:
```
$ agi 'github.com/openshift/node-observability-operator/pkg/operator/controller\"'
pkg/operator/controller/nodeobservability/daemonset.go
17:	opctrl "github.com/openshift/node-observability-operator/pkg/operator/controller"

pkg/operator/controller/nodeobservability/service.go
20:	opctrl "github.com/openshift/node-observability-operator/pkg/operator/controller"

pkg/operator/controller/nodeobservabilityrun/controller.go
47:	opctrl "github.com/openshift/node-observability-operator/pkg/operator/controller"
```

Reply:
Yes, because I also used opctrl in my PR after seeing this.